src/plugins/tables.test.md

```diff
@@ -0,0 +1,39 @@
+# Tables
+
+## Hashing
+
+```polygolf
+print (table_get
+    (table
+        ("▄ ▄▄▄" => "A")
+        ("▄▄▄ ▄ ▄ ▄" => "B")
+        ("▄▄▄ ▄ ▄▄▄ ▄" => "C")
+    )
+    (argv_get 0)
+);
+```
+
+```polygolf tables.testTableHashing(999)
+print (list_get (list "B" "C" "A") (((@FunctionCall (@BuiltinIdent "hash") (argv_get 0)):0..4294967295 mod 11) mod 3));
+```
+
+```polygolf tables.testTableHashing(9)
+print (list_get (list "A" "B" " " "C") (((@FunctionCall (@BuiltinIdent "hash") (argv_get 0)):0..4294967295 mod 9) mod 4));
+```
+
+## List lookup
+
+```polygolf
+table_get
```

Comment:
trolled again by forgetting that "pending review" means nobody can see my comment, i wrote this yesterday

Reply:
> does this rely on some testing-exclusive behavior? trying to compile this code on my machine produces `No variant could be compiled: Polygolf Type error. Operator 'table_get type error. Expected [(Table T1 T2), T1] but got [(Table (Text 5..11) (Ascii 1..1)), Text].`

Yes, testing is less strict about the input than the CLI - programs are not typechecked when testing plugins. However the CLI behaviour is correct since `Text` is not a subtype of `(Text 5..11)`.

Reply:
Right, gotcha